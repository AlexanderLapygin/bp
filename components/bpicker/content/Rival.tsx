import * as React from 'react'
import {Card} from "@material-ui/core";
import CardHeader from '@material-ui/core/CardHeader';
import CardContent from "@material-ui/core/CardContent";

type Props = {
  title: string,
  content: string,
  styles: any
};

export const Rival = (props: Props) => {
  return (
    <Card>
      <CardHeader title={props.title}/>
      <CardContent>
        {props.content}
      </CardContent>
    </Card>
  );
}
