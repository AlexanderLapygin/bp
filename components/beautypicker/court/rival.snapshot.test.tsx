import * as React from "react";
import * as TestRenderer from 'react-test-renderer';
import {Rival} from "./Rival";

it('renders correctly', () => {
  const tree = TestRenderer.create(<Rival/>).toJSON()
  expect(tree).toMatchSnapshot()
})
