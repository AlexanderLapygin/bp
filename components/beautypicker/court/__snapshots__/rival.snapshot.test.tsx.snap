// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
>
  <div
    className="MuiCardHeader-root"
  >
    <div
      className="MuiCardHeader-content"
    />
  </div>
  <div
    className="MuiCardContent-root"
  />
</div>
`;
